CHANGELOG.md

```diff
@@ -4,7 +4,8 @@
 
 ### Added
 
--  Added theme support for `UserMenuHeaderComponent`. 
+-  Added theme support for `<pxb-user-menu>`'s header. 
```

Comment:
I would keep the mention of the UserMenuHeaderComponent because we're currently already supporting the styles for the `<pxb-user-menu>` header.

Maybe something like: 'Added theme support for `UserMenuHeaderComponent` when used in a `<pxb-user-menu>`

Reply:
So there is actually a thing called `UserMenuHeaderComponent` in the code? Or was it styled `pxb-menu-header`? Will our end user see a thing called "UserMenuHeaderComponent"?

Reply:
Yep! `UserMenuHeaderComponent` is a brand new component to be released in next major version of @pxblue/angular-components.  It replaces the use of the `DrawerHeaderComponent` inside of a `UserMenu`.